dace/sdfg/state.py

```diff
@@ -303,7 +308,7 @@ def replace(self, name: str, new_name: str):
         :param name: Name to find.
         :param new_name: Name to replace.
         """
-        raise NotImplementedError()
+        pass
 
     @abc.abstractmethod
```

Comment:
Initially the idea was to never instantiate a ControlFlowBlock - only subclasses of it, each of which being a graph (States and Control flow regions). However, with the change to use break/return/continue blocks, we now have blocks being used which should NOT contain further elements - so are no longer a graph. That's why these methods now all of a sudden need to not throw NotImplementedErrors. Or rather, they could be left as throwing those errors, but then at each point where we iterate over nodes of a graph we would need to make sure to not try and iterate into a control flow block, only into states and regions etc.

So that's the background - and now why I left the abstractmethod decorator: It should still be the case that you never instantiate a BlockGraphView directly and always explicitly subclass it. In 90% of use cases, that would mean implementing these methods here, since you would usually be making a _graph_ type object.